[react-devtools-inline][4.22.0]: broken published package

## Steps To Reproduce

1. install `react-devtools-inline` in a project that has `react` and `react-is`
2. open node
3. `require('react-devtools-inline')`
4. See error:
```
Uncaught:
Error: Cannot find module '/Users/jstejada/code/jstejada-react/build/oss-experimental/react-is'
Require stack:
- /home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js
- <repl>
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.14 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:1344:18)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30)
    at Object.3 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:5710:17)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30)
    at Object.10 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:495:64)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30) 
```

## The current behavior

Published package contains the following code:

```
module.exports = require("/Users/jstejada/code/jstejada-react/build/oss-experimental/react-is");
...
module.exports = require("/Users/jstejada/code/jstejada-react/build/oss-experimental/react");
```

## The expected behavior

No absolute paths in bundle. successful evaluation.

--- a/packages/react-devtools-inline/webpack.config.js
+++ b/packages/react-devtools-inline/webpack.config.js
@@ -33,14 +33,6 @@ const babelOptions = {
   ),
 };
 
-const builtModulesDir = resolve(
-  __dirname,
-  '..',
-  '..',
-  'build',
-  'oss-experimental',
-);
-
 module.exports = {
   mode: __DEV__ ? 'development' : 'production',
   devtool: __DEV__ ? 'eval-cheap-source-map' : 'source-map',
@@ -57,10 +49,10 @@ module.exports = {
     libraryTarget: 'commonjs2',
   },
   externals: {
-    react: resolve(builtModulesDir, 'react'),
-    'react-dom': resolve(builtModulesDir, 'react-dom/unstable_testing'),
-    'react-is': resolve(builtModulesDir, 'react-is'),
-    scheduler: resolve(builtModulesDir, 'scheduler'),
+    react: 'react',
+    'react-dom': 'react-dom',
+    'react-is': 'react-is',
+    scheduler: 'scheduler',
   },
   node: {
     // source-maps package has a dependency on 'fs'

